modules/raft/src/integrationTest/java/org/apache/ignite/raft/server/JraftAbstractTest.java

```diff
@@ -201,7 +220,39 @@ protected JraftServerImpl startServer(int idx, Consumer<RaftServer> clo, Consume
 
         optionsUpdater.accept(opts);
 
-        JraftServerImpl server = TestJraftServerFactory.create(service, opts);
+        VaultManager vaultManager = new VaultManager(new PersistentVaultService(vaultPath(workingDir.basePath())));
+
+        vaultManagers.add(vaultManager);
+
+        assertThat(vaultManager.startAsync(new ComponentContext()), willCompleteSuccessfully());
+
+        Map<String, LogStorageFactory> logStorageFactoryByGroupName = Map.of(
+                PARTITION_GROUP_NAME, partitionsLogStorageFactory,
+                CmgGroupId.INSTANCE.toString(), partitionsLogStorageFactory,
+                MetastorageGroupId.INSTANCE.toString(), partitionsLogStorageFactory
+        );
+
+        Map<String, Path> serverDataPathByGroupName = Map.of(
+                PARTITION_GROUP_NAME, workingDir.basePath(),
+                CmgGroupId.INSTANCE.toString(), workingDir.basePath(),
+                MetastorageGroupId.INSTANCE.toString(), workingDir.basePath()
+        );
```

Comment:
Why do we need this 'partitions, metastorage, cmg' specifics in a test that is about Raft and nothing more?